source/_components/switch.enocean.markdown

```diff
@@ -13,19 +13,25 @@ ha_release: 0.21
 ha_iot_class: "Local Push"
 ---
 
-An EnOcean switch can take many forms. Currently only one type has been tested: Permundo PSC234
+An EnOcean switch can take many forms. Currently only two types have been tested and confirmed to work well: Eltako FSR61 and Permundo PSC234
 
 
 To use your EnOcean device, you first have to set up your [EnOcean hub](/components/enocean/) and then add the following to your `configuration.yaml` file:
 
+
 ```yaml
 # Example configuration.yaml entry
 switch:
   - platform: enocean
+    name: my_switch_name
     id: [0x01,0x90,0x84,0x3C]
+    sender_id: [0xff,0xc6,0xea,0x14]
+    subtype: fsr61
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.